highfidelity/hifi
# added Entities.findbyName

Adding feature to find entities by name as I've seen a lot of users do this kind of thing in scripts:

```
 Entities.findEntities(MyAvatar.position, SEARCH_RADIUS).forEach(function(element){
            var name = Entities.getEntityProperties(element, 'name').name;
		
            if (name === "Balloon Cage"){
                _this.balloonCage = element;
                print("found my balloonCage");
                // print(element);
                // print(name);
                return;
            }

        });
		
        if (_this.balloonCage === undefined){
            print("No Balloon Cage Found!");
        }			
```
This just cuts out having to do the for loop to see if those entities match by name

You can do the above now with: 
Entities.findEntitiesByName("Ballon Cage", MyAvatar.position, 30, true);

test plan:

1. create a box named "box-test"
2. open console and run Entities.findEntitiesByName("box-test", MyAvatar.position, 30, false);
3. check the results back is the id of box-test
4. create another box named "Box-Test"
5. run the same command in the console and see that you get back an array with two ids.
6. run Entities.findEntitiesByName("box-test", MyAvatar.position, 30, true);
7. Confirm that you only match the box with proper case
